cluster/selfplay/cc_wrapper.sh

```diff
@@ -33,6 +33,12 @@ GAMES=`gsutil ls "gs://$BUCKET_NAME/data/selfplay/$BASENAME/*.zz" | wc -l`
 
 gsutil cp $MODEL_FILE .
 
+GPU_TO_USE=`nvidia-smi | grep -o [0-9]*% | cut -d% -f1 | paste -s -d">" - | bc`
+
+export CUDA_VISIBLE_DEVICES=$GPU_TO_USE
```

Comment:
but why...? Is this some weirdness with the new NVidia Image that we're only nows seeing?

Reply:
Not sure why now.  In short, the cc-player can "see" both GPUs, and it doesn't automatically choose which GPU to put its ops on -- it always uses gpu0.  Setting the visible devices to either 0/1 makes that the only visible GPU to the binary.

the GKE docs suggest that a pod should only see the GPUs it requested in its spec, which is clearly not happening.  So i'd like to followup with vish about this.